### PR TITLE
Add Serbian locale

### PR DESCRIPTION
**Pull request by cyriaka90**
## Pull Request Checklist

Thank you for taking the time to improve Arrow! Before submitting your pull request, please check all *appropriate* boxes:

- [x] 🧪  Added **tests** for changed code.
- [x] 🛠️  All tests **pass** when run locally (run `tox` or `make test` to find out!).
- [x] 🧹  All linting checks **pass** when run locally (run `tox -e lint` or `make lint` to find out!).
- [ ] 📚  Updated **documentation** for changed code.
- [x] ⏩  Code is **up-to-date** with the `master` branch.

## Description of Changes

Adds Serbian locale. Serbian actually can be written with both Latin or Cyrillic script - I chose the Latin one here. Not sure if arrow could support both?

### CODE REVIEW

**Comment by krisfremen** [comment]
@cyriaka90 hvala lepo!

we have an ongoing issue to discuss how to implement native alphabets vs latin alphabet in #974

File: arrow/locales.py [hunk 1]
```diff
@@ -4985,3 +4985,97 @@ def _ordinal_number(self, n: int) -> str:
...
+        "januar",  # Јануар
+        "februar",  # фебруар
+        "mart",  # март
+        "april",  # април
+        "maj",  # мај
+        "juni",  # јун
+        "juli",  # јул
+        "avgust",  # август
+        "septembar",  # октобар
+        "oktobar",  # септембар
```
Comment (krisfremen) [inline arrow/locales.py:5027]: comments seem to be reversed :)

**Comment by cyriaka90** [comment]
> we have an ongoing issue to discuss how to implement native alphabets vs latin alphabet in #974

Thanks for the hint!

**Review by krisfremen — APPROVED** [review]
Looks good to me other than the small comment mixup!

File: arrow/locales.py (hunk 1 above)
Comment (cyriaka90) [reply arrow/locales.py:5027]: Oh, yes, of course - I'll change it! :D

**Comment by anishnya** [comment]
Merged in. Keep up the great work @cyriaka90!

**Comment by cyriaka90** [comment]
Thank you! :D

**Pull request merged**